app/routes/events/view/edit.js

```diff
@@ -17,7 +17,7 @@ export default class EditRoute extends Route.extend(AuthenticatedRouteMixin, Eve
 
     const event = this.modelFor('events.view');
     const { currentUser } = this.authManager;
-    if (!(currentUser.isAnAdmin || currentUser.email === event.owner.get('email') || event.organizers.includes(currentUser)
+    if (!(currentUser.isAnAdmin || (currentUser.email === event.owner.get('email') || !event.owner.get('email')) || event.organizers.includes(currentUser)
```

Comment:
Why was this change necessary

Reply:
when initially creating an event the owner email is null